dnsmasq/files/dnsmasq.init

```diff
@@ -455,6 +460,8 @@ dhcp_add() {
 		DNS_SERVERS="$DNS_SERVERS $dnsserver"
 	}
 
+	dnsmasq_device_list="${dnsmasq_device_list} $ifname"
+	dnsmasq_interface_list="${dnsmasq_interface_list} $net"
```

Comment:
Initialized at 1071, 1072
Populated in dhcp_add at 463, 464       
used at 1115, 1125